tests/modules/propagate.c

```diff
@@ -121,6 +121,64 @@ int propagateTestTimerNestedReplCommand(RedisModuleCtx *ctx, RedisModuleString *
     return REDISMODULE_OK;
 }
 
+void timerHandlerMaxmemory(RedisModuleCtx *ctx, void *data) {
+    REDISMODULE_NOT_USED(ctx);
+    REDISMODULE_NOT_USED(data);
+
+    RedisModuleCallReply *reply = RedisModule_Call(ctx,"SETEX","ccc!","timer-maxmemory-start","100","1");
+    RedisModule_FreeCallReply(reply);
+    reply = RedisModule_Call(ctx, "CONFIG", "ccc!", "SET", "maxmemory", "1");
```

Comment:
side note, i think i would be willing to leave this case as something that no sane module should be doing.
i.e. calling CONFIG SET maxmemory from a timer.

Reply:
do you want me to explicitly block it? or just assume no module would even do it?

if we want to block it, should it be in call() or in RM_Call?

Reply:
add test:
MULTI
INCR x
config set maxmemory 1
INCR x
exec

x should be 2

Reply:
for the record. we realized that it's not only modules, but also MULTI, and that the solution is probably to allow the CONFIG SET, but skip the eviction because we can't afford to propagate.

Reply:
yes